bin/Powheg/run_pwg.py

```diff
@@ -1174,17 +1175,16 @@ def runhnnlo(folderName, njobs, QUEUE):
                     test_pdf2 = n_column[1].strip()
 
             if not (test_pdf1 == test_pdf2) :
-                print "ERROR: PDF settings not equal for the 2 protons: "+test_pdf1+" vs "+test_pdf2+"... Please check your datacard"
-                quit()
+                raise RuntimeError("ERROR: PDF settings not equal for the 2 protons: {0} vs {1}... Please check your datacard".format(test_pdf1, test_pdf2))
```

Comment:
you have tested this, right? it saves one line but does exactly the same thing I suppose

Reply:
Yes, this has been tested in recent gridpack production